src/services/index.js

```diff
@@ -0,0 +1,67 @@
+const apiBase = 'https://api.themoviedb.org/3';
+
+const transformMovies = (movie) => (
+  {
+    isMain: true,
+    id: movie.id,
+    title: movie.title,
+    rating: movie.vote_average,
+    posterPath: movie.poster_path,
+    backdropPath: movie.backdrop_path,
+    runtime: movie.runtime,
+    genreIds: movie.genre_ids,
+    genres: movie.genres,
+    overview: movie.overview,
+  }
+);
+
+async function getResorce(url) {
+  const res = await fetch(`${apiBase}${url}`);
+  if (!res.ok) {
+    throw new Error(`Could not fetch ${url}, received ${res.status}`);
+  }
+  return res.json();
+}
+
+async function getMovies(type, pageNumber) {
+  const res = await getResorce(`/movie/${type}?api_key=95ba99e0f191e9eabd8a1d0c164f0af3&language=en-US&page=${pageNumber}`);
```

Comment:
This is training project and holding `api_key=95ba99e0f191e9eabd8a1d0c164f0af3` here is ok. But it is not secure. It's better put this api_key into env variables